### ISSUE TITLE
Recent update to HA broke browser_mod.  It can not play media (TTS) anymore.

### ISSUE THREAD

**Issue by Genzzry** [thomasloven/hass-browser_mod#322]
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
2022-05-04 23:39:03 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-05-04 23:39:03 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration adaptive_lighting which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-05-04 23:39:03 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration browser_mod which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-05-04 23:39:08 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548337475744] Received invalid command: browser_mod/update
2022-05-04 23:39:27 WARNING (Recorder) [homeassistant.components.recorder.migration] Database is about to upgrade. Schema version: 25
2022-05-04 23:39:27 WARNING (Recorder) [homeassistant.components.recorder.migration] Adding index `ix_statistics_runs_start` to database. Note: this can take several minutes on large databases and slow computers. Please be patient!
2022-05-04 23:39:29 ERROR (SyncWorker_0) [root] Uncaught exception
RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe
2022-05-04 23:39:57 WARNING (MainThread) [homeassistant.config_entries] Config entry 'CGNV4' for upnp integration not ready yet: ('', None); Retrying in background
2022-05-04 23:40:57 WARNING (MainThread) [homeassistant.helpers.service] Unable to find referenced entities media_player.pc_drone
2022-05-04 23:40:58 WARNING (MainThread) [homeassistant.helpers.service] Unable to find referenced entities media_player.pc_drone

**Comment by Genzzry** [comment]
Oddly, it started working again by itself after a few hours... but stopped working again when HA was restarted.

**Comment by Genzzry** [comment]
It seems to work again after navigating around Lovelace randomly for a while & turning devices like lights on/off... although I'm not sure on whats "fixing" it... but it breaks again after every reboot... and can be broken for a day or more before it decides to work again.

Here's some more logs:
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
2022-05-07 13:56:35 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-05-07 13:56:35 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration adaptive_lighting which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-05-07 13:56:35 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration browser_mod which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-05-07 13:56:38 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [547610959536] Received invalid command: browser_mod/connect
2022-05-07 13:56:39 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [547610959536] Received invalid command: browser_mod/update
2022-05-07 13:56:59 WARNING (MainThread) [homeassistant.helpers.service] Unable to find referenced entities media_player.pc_drone
2022-05-07 13:56:59 WARNING (MainThread) [homeassistant.helpers.service] Unable to find referenced entities media_player.pc_drone
2022-05-07 14:14:28 WARNING (MainThread) [homeassistant.helpers.service] Unable to find referenced entities media_player.pc_drone
2022-05-07 14:14:32 WARNING (MainThread) [homeassistant.helpers.service] Unable to find referenced entities media_player.pc_drone
